### PR TITLE
fix: fix timeline and topology view

### LINKED ISSUE #1788: [🐛 Bug]: Alert sidebar timeline indefinitely grows
The video best describes the problem - the timeline can indefinitely grow and affect the whole sidebar. Topology "grows" with it and there is no bottom margin.
Timeline should be with a fixed size, if it grows more then it, it should be scrollable.
Topology should be with a fixed size and some margin in the end.

https://github.com/user-attachments/assets/f078f86a-2412-459f-8aa1-b5745b17f740

### PR DESCRIPTION
**Pull request by 35C4n0r**
Closes #1788 
/claim #1788 

## ✅ Checks

- [ ] My pull request adheres to the code style of this project
- [ ] My code requires changes to the documentation
- [ ] I have updated the documentation as required
- [ ] All the tests have passed

## ℹ Additional Information

https://www.loom.com/share/ab2b738e098b47b5ad06220adeea81b8

### CODE REVIEW

File: keep-ui/app/alerts/alert-timeline.tsx [hunk 1]
```diff
@@ -76,7 +76,7 @@ const AlertTimeline: React.FC<AlertTimelineProps> = ({ alert, auditData, isLoadi
   ));
 
   return (
-    <div className="flex-grow">
+    <div className="h-[500px] overflow-y-auto">
```
Comment (talboren) [inline keep-ui/app/alerts/alert-timeline.tsx:79]:
```suggestion
    <div className="max-h-[500px] overflow-y-auto">
```

File: keep-ui/app/topology/topology.tsx [hunk 2]
```diff
@@ -183,7 +183,7 @@ const TopologyPage = ({
     );
 
   return (
-    <Card className="p-4 md:p-10 mx-auto h-full relative">
+    <Card className="p-4 md:p-10 mx-auto h-[800px] relative mb-10">
```
Comment (talboren) [inline keep-ui/app/topology/topology.tsx:186]:
```suggestion
    <Card className="p-4 md:p-10 mx-auto h-full relative mb-10">
```

**Review by talboren — APPROVED** [review]
LGTM! thank you :)

**Review by shahargl — APPROVED** [review]
lgtm

**Pull request merged**